src/scipp/core/like.py

```diff
@@ -23,15 +23,26 @@ def zeros_like(
     var: _Union[_cpp.Variable,
                 _cpp.DataArray]) -> _Union[_cpp.Variable, _cpp.DataArray]:
     """
-    Constructs a new object with the same dims, shape, unit and dtype as the input
-    (:class:`Variable` or :class:`DataArray`), but with all values initialized to 0.
+    Return a Variable or DataArray with the same dims, shape, unit, and dtype as the
+    input and all values initialized to 0.
+
     If the input has variances, all variances in the output are set to 0.
-    If the input is a :class:`DataArray`, coordinates and attributes are shallow-copied
+    If the input is a data array, coordinates and attributes are shallow-copied
     and masks are deep copied.
 
-    :param var: Input variable or data array.
+    Parameters
+    ----------
+    var:
+        Input object defining dims, shape, unit, and dtype of the output
+
+    Returns
+    -------
+    :
+        New object of zeros.
 
-    :seealso: :py:func:`scipp.zeros` :py:func:`scipp.ones_like`
+    See Also
+    --------
```

Comment:
I tend to dislike the rst things where the number of dashes has to match the length of the text above, but I guess in this case it's because the docstring also becomes the help string and this makes things more readable.

Would be nice if we could make html-rendered help strings ;-)

Reply:
This is unrelated to Napoleon, this is the Numpy style. It supports also Google style.

Reply:
Would be nice to have a tool that allows you to switch between styles in one go.
I found this https://github.com/dadadel/pyment/, but I'm a little suspicious, as I know how brittle string parsing can be...

I remember @arm61 having issues with the Google style when he had long sentences which would not indent properly if you tried to split them over several lines of code. He ended up putting some flake8 `noqa` statements for long lines everywhere...
If Napoleon converts the Google (and Numpy) style to sphinx style, then this means it would handle long lines better?